components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -108,11 +112,15 @@ class PromptFeature private constructor(
     onNeedToRequestPermissions: OnNeedToRequestPermissions
 ) : LifecycleAwareFeature, PermissionsFeature, Prompter {
     private var scope: CoroutineScope? = null
+    private var scope2: CoroutineScope? = null
```

Comment:
@Baron-Severin let's give these scopes better names :)

Reply:
Whoops!  I meant to comment there too...